scout/server/blueprints/variants/static/form_scripts.js

```diff
@@ -1,12 +1,26 @@
 function populateCytobands(cytobands){
+  var chromPosPattern = new RegExp("^(?:chr)?([1-9]|1[0-9]|2[0-2]|X|Y|MT)$");
   var chrom = document.forms["filters_form"].elements["chrom"].value;
-  if(chrom===""){
+  var chromPos = document.forms["filters_form"].elements["chrom_pos"].value;
+  var chromosome = "";
+  console.log("Populate cytobands")
+  var matchedChrName = chromPos.match(chromPosPattern)
+  if(chrom==="" && chromPos===""){
     startElem.value = "";
     endElem.value = "";
     return //only reset cytoband select element
+  // Set the selected chromosome name
+  } else if (chrom !== "" && matchedChrName !== null){
```

Comment:
I'm so glad we didn't write the entire app in js.. === && !==, !== && === and == && ==..